### PR TITLE
ECEP-113: Style News & Related Patterns

### PR DESCRIPTION
**Pull request by wesleyboar**
### Overview

Make changes to support ECEP News.

### Related

- [ECEP-113](https://jira.tacc.utexas.edu/browse/ECEP-113)
- required by https://github.com/TACC/Core-CMS/pull/466
- required by https://github.com/TACC/Core-CMS-Resources/pull/127

### Changes

- style figures and captions
- style bootstrap pagination
- move blog css to [TACC/Core-CMS](https://github.com/TACC/Core-CMS)
- style more distance between paragraphs
- style updates for offset content pattern

### Screenshots & Testing

See https://github.com/TACC/Core-CMS/pull/466.

### Notes

<details><summary>This PR was broken into smaller ones to track isolated changes separately.</summary>

- 847d0e2157d8eadd0437fb1de69fe720db031cb9
- https://github.com/TACC/Core-Styles/pull/12
- https://github.com/TACC/Core-Styles/pull/13
- https://github.com/TACC/Core-Styles/pull/14
- https://github.com/TACC/Core-Styles/pull/15
- de8f5400d5e9fb2cd2a8bfcb502ad62cc267e5c3

</details>

### CODE REVIEW

File: source/_imports/components/django.cms.post.css [hunk 1]
```diff
@@ -1,11 +0,0 @@
...
-Django CMS Blog Post Components
-
-Components for [Django CMS Blog Post](https://djangocms-blog.readthedocs.io) UI. These classes are authored in templates of third-party origin, and expanded by us, but they have no third-party styles.
-
-Reference:
-
-- [Django CMS Blog `.post-` Class Components](https://github.com/TACC/CORE-cms/tree/main/taccsite_cms/templates/djangocms_blog)
-
-Styleguide Components.DjangoCMS.Post
-*/
```
Comment (wesleyboar) [inline source/_imports/components/django.cms.post.css:11]: (merged file into [TACC/Core-CMS:/taccsite_cms/static/site_cms/css/src/…](https://github.com/TACC/Core-CMS/blob/task/ECEP-113-support-news/taccsite_cms/static/site_cms/css/src/_imports/components/django.cms.blog.css) which then split its styles into multiple stylesheets)

File: source/_imports/objects/o-offset-content.css [hunk 2]
```diff
@@ -24,10 +32,14 @@ Styleguide Objects.OffsetContent
...
+  .figure:is(:--o-offset-content) .figure-caption {
+    padding-right: 125px;
+  }
 }
 @media only screen and (--medium-and-above) and (--max-wide-and-below) {
-  [class*="o-offset-content--"] {
-    max-width: 50%;
+  :--o-offset-content {
+    max-width: 560px;
   }
```
Comment (wesleyboar) [inline source/_imports/objects/o-offset-content.css:43]: <details><summary>The offset content changes support the use of `c-offset-content--right` on many blog posts.</summary>

- https://prod.ecep.tacc.utexas.edu/news/2021/02/25/cape-framework-published-communications-acm/
- https://prod.ecep.tacc.utexas.edu/news/2019/09/13/2019-state-computer-science-education-report-released/
- https://prod.ecep.tacc.utexas.edu/news/2017/11/15/ecep-alliance-shares-commitment-csforall-summit/
- https://prod.ecep.tacc.utexas.edu/news/2017/08/31/puerto-rico-set-teach-exploring-computer-science-spanish/
- https://prod.ecep.tacc.utexas.edu/news/2017/07/26/ecep-website-redesign-focus-access-content-and-design/
- https://prod.ecep.tacc.utexas.edu/news/2017/05/04/nv-nh-nc-pr-awarded-mini-grants-further-cs-education-equity/
- https://prod.ecep.tacc.utexas.edu/news/2017/02/01/rethinking-perkins-expand-access-k-12-computer-science/

</details>

**Review by wesleyboar — COMMENTED** [review]
notes for reviewers

File: source/_imports/elements/html-elements.css [hunk 3]
```diff
@@ -110,6 +110,10 @@ Reference:
 Styleguide Elements.TextContent
 */
 
+p {
+  margin-bottom: 2rem; /* overwrite Bootstrap */
+}
+
```
[the remark this answers is not in the document]
Comment (taoteg) [reply source/_imports/elements/html-elements.css:116]: Seems reasonable to leave it here until you are sure. =)

**Review by taoteg — APPROVED** [review]
LGTM (as I review the other 2 related PRs).

File: source/_imports/components/bootstrap.pagination.css [hunk 4]
```diff
@@ -0,0 +1,29 @@
+/*
+Pagination (Bootstrap)
+
+Style Bootstrap pagination. See:
+
+- [Bootstrap Pagination](https://getbootstrap.com/docs/4.0/components/pagination/)
+
+Styleguide Components.Bootstrap.Pagination
+*/
+@import url("@tacc/core-styles/source/_imports/components/c-page.css");
```
Comment (wesleyboar) [inline source/_imports/components/bootstrap.pagination.css:10]: **Hint:** The [imported stylesheet](https://github.com/TACC/Core-Styles/blob/main/source/_imports/components/c-page.css) was added in https://github.com/TACC/Core-Styles/pull/5.

**Comment by wesleyboar** [comment]
All related PR have been approved. I merge now.

**Pull request merged**